### PR TITLE
Constraint to enforce stackdriver monitoring on a GKE cluster

### PR DESCRIPTION
**Pull request by charliewolf**

### CODE REVIEW

**Comment by charliewolf** [comment]
per @jberlinsky CLA re-check

**Comment by charliewolf** [comment]
I signed it!

File: policies/templates/gcp_gke_enable_stackdriver_monitoring_v1.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Check to see if GKE stackdriver monitoring is enabled
+# https://cloud.google.com/monitoring/kubernetes-engine/
+
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: gcp-gke-disable-legacy-endpoints-v1
```
Comment (joecheuk) [inline policies/templates/gcp_gke_enable_stackdriver_monitoring_v1.yaml]: The name is referring to another policy.

File: samples/gke_enable_stackdriver_monitoring.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPGKEEnableStackdriverMonitoringConstraintV1
+metadata:
+  name: enable_gke_stackdriver_monitoring
+  annotations:
+    benchmark: GKE_HARDENING_GUIDELINE
```
Comment (joecheuk) [inline samples/gke_enable_stackdriver_monitoring.yaml]: This is part of CIS, can you change this to CIS11_7.02 and add another description field to describe the benchmark? e.g. https://github.com/forseti-security/policy-library/blob/master/samples/iam_blacklist_role.yaml

File: validator/test/fixtures/gke_enable_stackdriver_monitoring/constraints/enable_gke_stackdriver_monitoring/data.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPGKEEnableStackdriverMonitoringConstraintV1
+metadata:
+  name: enable_gke_stackdriver_monitoring
+  annotations:
+    benchmark: GKE_HARDENING_GUIDELINE
```
Comment (joecheuk) [inline validator/test/fixtures/gke_enable_stackdriver_monitoring/constraints/enable_gke_stackdriver_monitoring/data.yaml]: This is part of CIS, can you change this to CIS11_7.02 and add another description field to describe the benchmark? e.g. https://github.com/forseti-security/policy-library/blob/master/samples/iam_blacklist_role.yaml

**Pull request merged**